extensions/wikia/PageHeader/PageTitle.class.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Wikia\PageHeader;
+
+use \RequestContext;
+use \Title;
+use \WikiaGlobalRegistry;
+use \WikiaApp;
+use \WikiaPageType;
+
+class PageTitle {
+
+	/* @var string */
+	public $title;
+
+	/* @var string */
+	public $prefix;
+
+	/* @var WikiaGlobalRegistry */
+	private $wg;
+
+	/* @var int */
+	private $namespace;
+
+	/* @var Title */
+	private $MWTitle;
+
+	const PREFIX_LESS_NAMESPACES = [ NS_MEDIAWIKI, NS_TEMPLATE, NS_CATEGORY, NS_FILE ];
+
+	/**
+	 * @param WikiaApp $app
+	 */
+	public function __construct( $app ) {
+		$this->wg = $app->wg;
+		$this->MWTitle = RequestContext::getMain()->getTitle();
```

Comment:
Why not `$app->wg->title`? That's what we use for `Counter`. Should we switch to this one, instead?